config/presets/testnet.go

```diff
@@ -112,12 +112,15 @@ func testnet() config.Config {
 			BeaconSyncWeightUnits:    800,
 		},
 		POET: activation.PoetConfig{
-			PhaseShift:        12 * time.Hour,
-			CycleGap:          2 * time.Hour,
-			GracePeriod:       10 * time.Minute,
-			RequestTimeout:    550 * time.Second, // RequestRetryDelay * 2 * MaxRequestRetries*(MaxRequestRetries+1)/2
-			RequestRetryDelay: 5 * time.Second,
-			MaxRequestRetries: 10,
+			// RequestTimeout = RequestRetryDelay * 2 * MaxRequestRetries*(MaxRequestRetries+1)/2
+			RequestTimeout:         550 * time.Second,
+			RequestRetryDelay:      5 * time.Second,
+			MaxRequestRetries:      10,
+			PhaseShift:             12 * time.Hour,
+			CycleGap:               2 * time.Hour,
+			GracePeriod:            10 * time.Minute,
+			FetchProofTimeout:      550 * time.Second,
+			SubmitChallengeTimeout: 10 * time.Minute,
```

Comment:
Why is this not equal to `RequestTimeout`?

Reply:
https://github.com/spacemeshos/go-spacemesh/pull/6136#discussion_r1680873895